frontend/src/audios.html

```diff
@@ -1,37 +1,37 @@
 ---
 pagination:
-  data: browse
+  data: audios
   size: 1
-  alias: lecture
-permalink: "/lecture/{{ lecture.objectID | slug }}/"
+  alias: audios
```

Comment:
I think it's better to address each item as `audio`, not `audios`. As it is only one for each page.

Reply:
Also, since this page represent a single audio, it is better to name the file as `audio.html`. When we introduce the home page, the audio search page will be called `audios.html`.

Reply:
file name and variable name changed to `audio`